js/src/attribute-mapping/attribute-mapping-delete-rule-modal.js

```diff
@@ -21,15 +50,19 @@ const AttributeMappingDeleteRuleModal = ( { onRequestClose = noop, rule } ) => {
 
 		try {
 			await deleteMappingRule( rule );
-			onRequestClose();
+			recordEvent( 'gla_attribute_mapping_delete_rule', {
+				context: 'attribute-mapping-delete-rule-modal',
+				...rule,
+			} );
```

Comment:
Same doubt as https://github.com/woocommerce/google-listings-and-ads/pull/1777#discussion_r1026220206.